core/src/main/resources/db/changelog/changesets/2024_11_22_hide_navbar.yaml

```diff
@@ -0,0 +1,9 @@
+databaseChangeLog:
+  - changeSet:
+      id: "hide_navbar"
+      author: mbemis
+      changes:
+        - addColumn:
+            tableName: html_page
+            columns:
+              - column: { name: hide_navbar, type: boolean, defaultValueBoolean: false }
```

Comment:
let's make this column non-nullable--that's best practice for columns that are mapped to `boolean` rather than `Boolean`